static/styles/popovers.scss

```diff
@@ -354,3 +355,11 @@ ul {
     opacity: 0.8;
     padding: 2px 0px 3px 0px;
 }
+
+.compose_actions_popover {
```

Comment:
I tihnk we can just match/reuse the gear menu box-shadow for this?

Reply:
The gear menu is the one on the top-right, right? I used the box-shadow CSS that was used for that dropdown-menu.

Reply:
Interesting, ok. I remember it being darker, but that could just be because it was on a more-white background. 

If possible it would be great to have it be literally the same CSS (so that changes in one place will get reflected in the other), but if that's complicated then no worries.

Reply:
It was the wrong box-shadow that I used. The one that is used for the gear menu is the one `bootstrap.css` has for the `.dropdown-menu` class. But, I used the one for `.nav .dropdown-menu` from `zulip.scss`. I could directly use the `.dropdown-menu` class, but I have made the changes here.

This is what it looks like now:
![Screenshot_5](https://user-images.githubusercontent.com/25329595/61189011-b8fdd100-a6a4-11e9-9552-96eaa24484a5.png)